Add documentation for series copy, optional series and draft courses (#598)
dodona-edu/dodona-edu.github.io: nl/guides/teachers/exercise-series-management/index.md

![reeksen beheren](./staff.course_manage_series_button.png)

Op deze pagina vind je rechtsboven de knoppen `Bestaande reeks kopiëren` en `Reeks aanmaken`.

![reeks aanmaken](./staff.course_new_series_button.png)

> Bij het kopiëren van een bestaande reeks selecteer je dan de reeks die je wilt kopiëren. Dit wordt standaard gefilterd op de huidige cursus, maar je kan reeksen kiezen van elke cursus waarvan je beheerder bent.

Reviewer: ```suggestion
Bij het kopiëren van een bestaande reeks selecteer je de reeks die je wil kopiëren. Dit wordt standaard gefilterd op de huidige cursus, maar je kan reeksen kiezen uit elke cursus die je beheert.
```

> ![reeks kopïeren](staff.series_new_copy.png)

Reviewer: ```suggestion
![reeks kopiëren](./staff.series_new_copy.png)
```

Je wordt dan doorgestuurd naar een formuler waar je de eigenschappen van de reeks kan instellen. Dit is ook het formulier waar je op terechtkomt als je een nieuwe reeks aanmaakt.
Je komt op een formulier terecht waarin je de volgende eigenschappen van de reeks kan instellen:

![series form](./staff.series_new.png)

> Eerst kies je de soort reeks je wil aanmaken. Je kan ofwel een normale reeks aanmaken, of een optionele reeks. De normale reeks heeft een duidelijke voortgangsaanduiding. Een optionele reeks wordt standaard ingeklapt voor je studenten, and maakt duidelijk dat de oefeningen in de reeks optioneel zijn.

Reviewer: ```suggestion
Eerst kies je de soort reeks die je wil aanmaken. Je kan ofwel een normale reeks aanmaken, of een optionele reeks. De normale reeks heeft een duidelijke voortgangsaanduiding. Een optionele reeks wordt standaard ingeklapt voor je studenten, en maakt duidelijk dat de oefeningen in de reeks optioneel zijn.
```

> De eerste keuze om te maken is wat voor soort reeks je wil aanmaken. Je kan ofwel een normale reeks aanmaken of een optionele reeks. Een optionele reeks wordt standaard ingeklapt voor je studenten, and maakt duidelijk dat de oefeningen in de reeks optioneel zijn. De normale reeks heeft een duidelijke voortgangsaanduiding.

Reviewer: ```suggestion
Eerst kies je de soort reeks je wil aanmaken. Je kan ofwel een normale reeks aanmaken, of een optionele reeks. De normale reeks heeft een duidelijke voortgangsaanduiding. Een optionele reeks wordt standaard ingeklapt voor je studenten, and maakt duidelijk dat de oefeningen in de reeks optioneel zijn.
```

Nadien stel je de volgende eigenschappen in:

* `Naam`: De naam van de oefeningenreeks. Binnen een cursus kunnen verschillende oefeningenreeksen dezelfde naam hebben, maar het is aangeraden om alle oefeningenreeksen een unieke naam te geven.

* `Deadline`: Een optionele deadline die aangeeft tot wanneer er rekening gehouden wordt met oplossingen die ingediend worden voor oefeningen van deze oefeningenreeks. 

>   Studenten kunnen na de deadline onbeperkt oplossingen blijven indienen voor oefeningen uit de oefeningenreeks en blijven daar nog steeds een beoordeling en feedback voor ontvangen. 

Reviewer: ```suggestion
  Studenten kunnen na de deadline onbeperkt oplossingen blijven indienen voor oefeningen uit de oefeningenreeks, en ontvangen daar ook nog steeds een beoordeling en feedback voor. 
```

  Je kan eenvoudig een nieuwe geheime link genereren door op de vernieuwknop te klikken. Dit kan je bijvoorbeeld doen als je per ongeluk de link kenbaar hebt gemaakt aan iemand die ze niet zou mogen zien. Hou er wel rekening mee dat de oude link niet meer zal werken van zodra je een nieuwe genereert.
  ![reekslink vernieuwen](./staff.series_hidden_link_reset.png)

* **Geavanceerde instellingen**:

  * `Verberg de "Voortgang groep" visualisatie voor studenten`: Bij een oefening wordt de voortgang van alle gebruikers in de cursus getoond. Hierin kan je zien hoeveel studenten een oefening reeds hebben begonnen of afgewerkt. Deze visualisatie is een meerwaarde voor studenten, maar je wil ze misschien uitschakelen voor examens.

  * `Verberg leeractiviteiten`: Als de leeractiviteiten verborgen zijn, dan zal enkel de beschrijving van deze reeks getoond worden. Je kan deze instelling gebruiken om bijvoorbeeld de leerstof van de reeks vooraf beschikbaar te stellen, zonder de oefeningen en leesactiviteiten al vrij te geven.

>   * `Nummer leeractiviteiten`: Als deze instelling actief is, krijgt elke oefening en leesactiveit een nummer, beginnende bij 1.

Reviewer: ```suggestion
  * `Nummer leeractiviteiten`: Als deze instelling actief is, krijgt elke oefening en leesactiviteit een nummer, beginnende bij 1.
```

>     Dit nummer wordt getoond in titels en oplijstingen en kan het makkelijke maken om over deze activiteiten te communiceren.

Reviewer: ```suggestion
    Dit nummer wordt getoond in titels en oplijstingen en kan het eenvoudiger maken om over deze activiteiten te communiceren.
```

Om de reeks aan te maken klik je op de knop `Reeks aanmaken` onderaan de pagina. De nieuwe oefeningenreeks wordt zo aan je cursus toegevoegd.

> Na het aanmaken van de reeks kom je op de pagina voor het beheren van de oefeningen in een reeks terecht.

Reviewer: ```suggestion
Na het aanmaken van de reeks kom je op de pagina terecht waar je de oefeningen van de reeks beheert.
```
